### ISSUE TITLE
Modifications to the underlying range input's min/max/step attributes are ignored

### ISSUE THREAD

**Issue by adatta02** [freqdec/fd-slider#33]
If you programatically update the min, max, or step attributes of the original input they aren't reflected in the shim'ed element.

See http://twitlabs.net/fd-slider/ if you test in a browser where the native element is available, when the max is updated you'll notice the slider handle jumps to the appropriate position. With the shim, nothing happens and the new max isn't reflected in the UI.

**Comment by freqdec** [comment]
Hi, I've just updated the code to enable this functionality on an opt-in basis. I feel that the DOMAttrMod event is so heavyweight that it just wouldn't be good karma if the choice wasn't at least offered. 

I've added a configuration option "watchAttributes" that instructs the script to set the appropriate "propertychange" or "DOMAttrModified" onthe input. Of course, setting this option to true will still not guarantee a full cross-browser solution as "propertychange" and "DOMAttrModified" will only be fired on Trident and Gecko respectively (but as these are the only two browsers that really need the shim you have most of the bases covered - Incidentally, Firefox landed support for the range element in the latest nightly; good news for all!).

I've also added a function "rescanAttributes" and set a data- attribute named "fd-range-enabled" on the input element to enable front end code to detect if the input has an associated shim or not - so if you _don't_ want to opt-in to using the events and prefer to use Javascript (which you are using to update the attributes anyway) then the following is possible:

if(inp.getAttribute("fd-range-enabled") == 1) {
    fdSlider.rescanAttributes();
};

Hopefully this is a solution that suits everybody. Feel free to speak up and many apologies for taking so dam long to get round to doing this.
